nautilus_open_any_terminal/nautilus_open_any_terminal.py

```diff
@@ -274,10 +276,6 @@ def get_menu_items(file: FileManager.FileInfo, callback, *, foreground: bool, te
         LOCAL_LABEL = _("Open {} Here")
         LOCAL_TIP = _("Open {} In This Directory")
 
-    # Let wezterm handle opening a local terminal
-    if terminal == "wezterm" and flatpak == "off":
-        return items
-
```

Comment:
Oh I see. I actually didn't realize wezterm had a nautilus extension. I am using wezterm locally compiled and installed under ~/.local/bin and didn't have wezterm's own extension installed.

After trying out the extension wezterm comes with, I do prefer nautilus-open-any-terminal over it, especially since with the other change in this PR I can configure to open in new tab which isn't an option in wezterm's own extension. But I will open a separate issue on this.

I've reverted these changes and limited this PR to just adding support for opening a new tab in wezterm.